assemblyline_v4_service/updater/client.py

```diff
@@ -136,13 +136,19 @@ def add_update_many(self, source: str, sig_type: str, data: List[Union[dict, Sig
             if isinstance(d, SignatureModel):
                 d = d.as_primitives()
 
+            # Compute the expected signature ID
+            sig_id = f"{sig_type}_{source}_{d['signature_id']}"
+            sig_exists = self.datastore.signature.get_if_exists(sig_id, as_obj=False)
```

Comment:
```suggestion
            sig_exists: SignatureModel = self.datastore.signature.get_if_exists(sig_id, as_obj=False)
```

Reply:
?

Reply:
`as_obj=False` will yield a dict (if the document exists), not a SignatureModel